crates/sim2h_server/src/main.rs

```diff
@@ -54,16 +55,18 @@ fn main() {
         });
     }
 
-    loop {
-        let result = sim2h.process();
-        if let Err(e) = result {
-            if e.to_string().contains("Bind error:") {
-                println!("{:?}", e);
-                exit(1)
-            } else {
-                error!("{}", e.to_string())
+    for _i in 0..num_cpus::get() {
```

Comment:
... this doesn't start any threads or anything, and the infinite loop inside cannot exit. `_i` will never advance beyond `0`.

Reply:
Woops, I blame lack of sleep? Fixed.